desertbot/desertbot.py

```diff
@@ -117,12 +117,12 @@ def irc_JOIN(self, prefix, params):
 
         message = IRCMessage('JOIN', prefix, channel, u'', self)
 
-        if message.User.Name == self.nickname:
```

Comment:
These changes are not right, it's still User and ReplyTo in the IRCMessage class